src/node/historical_queries_adapter.cpp

```diff
@@ -189,13 +196,112 @@ namespace ccf::historical
     return HistoricalTxStatus::Valid;
   }
 
+  std::optional<ServiceInfo> find_previous_service_identity(
+    auto& ctx,
+    ccf::historical::StatePtr& state,
+    AbstractStateCache& state_cache)
+  {
+    SeqNo target_seqno = state->transaction_id.seqno;
+    LOG_DEBUG_FMT("target seqno: {}", target_seqno);
```

Comment:
Remove these debug lines, or make them `LOG_TRACE_FMT` maybe?

Reply:
Good point, most of those shouldn't be necessary anymore.